docs/streaming-flume-integration.md

```diff
@@ -129,6 +138,12 @@ configuring Flume agents.
 		JavaReceiverInputDStream<SparkFlumeEvent>flumeStream =
 			FlumeUtils.createPollingStream(streamingContext, [sink machine hostname], [sink port]);
 	</div>
+	<div data-lang="python" markdown="1">
```

Comment:
I updated the doc. Because there is no a Python example for `FlumeUtils.createPollingStream` (it requires the user installing the Spark sink jar to Flume, so we cannot write an out-of-the-box example), I just copied the description of the encoding function to here.